policy/modules/system/logging.te

```diff
@@ -607,9 +607,10 @@ files_read_kernel_symbol_table(syslogd_t)
 files_var_lib_filetrans(syslogd_t, syslogd_var_lib_t, { file dir })
 
 fs_getattr_all_fs(syslogd_t)
-fs_read_efivarfs_files(syslogd_t)
 fs_search_auto_mountpoints(syslogd_t)
 fs_search_cgroup_dirs(syslogd_t)
+fs_read_efivarfs_files(syslogd_t)
+read_files_pattern(syslogd_t, init_var_run_t, init_var_run_t)
```

Comment:
There is an interface for accessing types outside the module as mentioned previously.

Reply:
I don't understand what that means. If it should be written in a different way, just say how.

Reply:
`init_read_pid_files(syslogd_t)` — is that it?

Reply:
Yes, I elaborated on that in the review a week ago.